src/store/chat/saga.ts

```diff
@@ -125,6 +135,8 @@ export function* setWhenUserJoinedRoom(conversationId: string) {
 }
 
 export function* performValidateActiveConversation(activeConversationId: string) {
+  yield put(setIsJoiningConversation(true));
```

Comment:
Why do we need to set these again in this saga? Nothing else should be calling this aside from the `validateActiveConversation` saga that's controlling the primary flow, right?